website/docs/best-practices/custom-generic-tests.md

```diff
@@ -9,7 +9,7 @@ hoverSnippet: Learn how to write your own custom generic data tests.
 dbt ships with [Not Null](/reference/resource-properties/data-tests#not-null), [Unique](/reference/resource-properties/data-tests#unique), [Relationships](/reference/resource-properties/data-tests#relationships), and [Accepted Values](/reference/resource-properties/data-tests#accepted-values) generic data tests. (These used to be called "schema tests," and you'll still see that name in some places.) Under the hood, these generic data tests are defined as `test` blocks (like macros).
 
 :::info
-There are tons of generic data tests defined in open source packages, such as [dbt-utils](https://hub.getdbt.com/dbt-labs/dbt_utils/latest/) and [dbt-expectations](https://hub.getdbt.com/calogica/dbt_expectations/latest/) — the test you're looking for might already be here!
+There are tons of generic data tests defined in open source packages, such as [<Constant name="dbt" />-utils](https://hub.getdbt.com/dbt-labs/dbt_utils/latest/) and [<Constant name="dbt" />-expectations](https://hub.getdbt.com/calogica/dbt_expectations/latest/) — the test you're looking for might already be here!
```

Comment:
Same question here - what if the proper name changes but this package is still called `dbt-utils`